spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkFilters.java

```diff
@@ -219,4 +224,9 @@ private static Expression handleEqual(String attribute, Object value) {
       return equal(attribute, convertLiteral(value));
     }
   }
+
+  private static String unquote(String attributeName) {
```

Comment:
Is this something that we should do for all Spark versions? Or is this specific to 3.2 and later?

Reply:
I think it applies to all 3.x versions.